### PR TITLE
Add support for Loki

### LINKED ISSUE #125: Support Grafana Loki for viewing logs
as an alternative to the existing Kibana integration, we should add Loki to the LinkingService. Make it configurable to switch between the two

URL format for Loki: 
`https://grafana.example.com/explore?orgId=1&left=["now-2d","now","loki",{"expr":"{app=\"my-app\"}"}]`

### PR DESCRIPTION
**Pull request by disrupted**
Closes #125

### CODE REVIEW

File: backend/streams_explorer/defaultlinker.py [hunk 1]
```diff
@@ -66,6 +69,8 @@ def get_redirect_streaming_app(
     ) -> Optional[str]:
         if link_type == "kibanalogs":
             return f"{settings.kibanalogs.url}/app/discover#/?_a=(columns:!(message),query:(language:lucene,query:'kubernetes.labels.app: \"{k8s_app.name}\"'))"
+        elif link_type == "loki":
+            return f'{settings.loki.url}/explore?orgId=1&left=["now-2d","now","loki",{{"expr":"{{app="{k8s_app.name}"}}"}}]'
```
Comment (philipp94831) [inline backend/streams_explorer/defaultlinker.py:73]: lets do now-1h as default start

Comment (philipp94831) [reply backend/streams_explorer/defaultlinker.py:73]: You are missing backslash `app=\"{k8s_app.name}\"`

File: backend/streams_explorer/defaultlinker.py [hunk 2]
```diff
@@ -66,6 +69,8 @@ def get_redirect_streaming_app(
     ) -> Optional[str]:
         if link_type == "kibanalogs":
             return f"{settings.kibanalogs.url}/app/discover#/?_a=(columns:!(message),query:(language:lucene,query:'kubernetes.labels.app: \"{k8s_app.name}\"'))"
+        elif link_type == "loki":
+            return f'{settings.loki.url}/explore?orgId=1&left=["now-1d","now","loki",{{"expr":"{{app=\\"{k8s_app.name}\\"}}"}}]'
```
Comment (philipp94831) [inline backend/streams_explorer/defaultlinker.py:73]: now-1h, not now-1d :-)

Comment (disrupted) [reply backend/streams_explorer/defaultlinker.py:73]: ah, my bad

**Review by philipp94831 — APPROVED** [review]

File: backend/streams_explorer/defaultlinker.py [hunk 3]
```diff
@@ -39,6 +36,12 @@ def __init__(self):
         if settings.kowl.enable:
             self.add_message_provider("kowl")
 
+        if settings.kibanalogs.enable:
+            self.add_logging_provider("Kibana Logs", "kibanalogs")
```
Comment (philipp94831) [inline backend/streams_explorer/defaultlinker.py:40]: Ah, just came to my mind now: We could also unify the label here, as we do for Kowl/AKHQ. Just "Logs" would be fine I guess. Up to you

Comment (disrupted) [reply backend/streams_explorer/defaultlinker.py:40]: agreed, if at some point we'd want to allow multiple services for the same task we can revisit this but for now I like the shorter and more uniform name

**Review by philipp94831 — APPROVED** [review]

**Pull request merged**